### PR TITLE
Add separate squad2.0 task for #969

### PR DESCRIPTION
**Pull request by theSage21**
Differences from squad

- If answering the question is impossible, answer is an empty string as per the dataset.
- `plausible_answers` key is populated in the event of an impossible question.

Everything else remains the same as the squad 1.1 task

### CODE REVIEW

File: parlai/tasks/squad2/agents.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+    def get(self, episode_idx, entry_idx=None):
+        article_idx, paragraph_idx, qa_idx = self.examples[episode_idx]
+        article = self.squad[article_idx]
+        paragraph = article['paragraphs'][paragraph_idx]
+        qa = paragraph['qas'][qa_idx]
+        context = paragraph['context']
+        question = qa['question']
+
+        answers = ['']
```
Comment (alexholdenmiller) [inline parlai/tasks/squad2/agents.py]: here we'll want to start with an empty list, as `'' in 'any string'` returns true, which would mark every sentence as containing the label in this case

File: parlai/tasks/squad2/agents.py [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+            context = context.replace(edited_answers[i], answers[i])
+
+        labels = []
+        label_starts = []
+        for sentence in sentences:
+            for answer in answers:
+                if answer in sentence and sentence not in labels:
+                    labels.append(sentence)
+                    label_starts.append(context.index(sentence))
+                    break
```
Comment (alexholdenmiller) [inline parlai/tasks/squad2/agents.py]: then down here you can do:
```
if len(labels) == 0:
    labels.append('')
```

**Comment by alexholdenmiller** [comment]
took a quick look with the following and it looked good:
```
import parlai.tasks.squad2.agents as s2
agents = [c for s, c in s2.__dict__.items() if s.endswith('Teacher') and 'squad2' in str(c)]
from parlai.core.params import ParlaiParser
p = ParlaiParser()
p.set_params(task='squad2')
opt = p.parse_args()
print('\n\n'.join([str(a(opt).act()) for a in agents]))
```

**Comment by muni2773** [comment]
@theSage21 @alexholdenmiller I understand the hit but I think the performance suffers because the underlying drqa NN is still built for Squad 1.1. Thoughts ?

**Pull request merged**